popUp.css

```diff
@@ -0,0 +1,212 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+.project-detail-card {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  overflow-x: hidden;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+
+  /* overflow-y: scroll; */
+  margin: 0 auto;
+  height: 100%;
+  width: 80%;
+
+  /* height: 1163px;
+  width: 1156px;
+  left: 0;
+  top: 0;
+  border-radius: 16px; */
```

Comment:
[OPTIONAL] Please don't leave any uncommented code in your codebase. So your codebase can look tidy and professional.